### PR TITLE
docs: refine verified fetch docs

### PR DESCRIPTION
**Pull request by 2color**
## Title

Refine the verified-fetch README

## Change checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/verified-fetch/src/index.ts [hunk 1]
```diff
@@ -140,15 +140,15 @@
...
+ * IPFS supports several data formats (typically referred to as codecs) which are included in the CID. `@helia/verified-fetch` attempts to abstract away some of the details for easier consumption.
  *
  * #### DAG-PB
  *
  * [DAG-PB](https://ipld.io/docs/codecs/known/dag-pb/) is the codec we are most likely to encounter, it is what [UnixFS](https://github.com/ipfs/specs/blob/main/UNIXFS.md) uses under the hood.
  *
  * ##### Using the DAG-PB codec as a Blob
  *
- * ```TypeScript
+ * ```typescript
```
Comment (achingbrain) [inline packages/verified-fetch/src/index.ts:151]: For why?  It's spelt [TypeScript](https://www.typescriptlang.org/).

Comment (2color) [reply packages/verified-fetch/src/index.ts:151]: Neither is my opinion. I just updated for consistency (some were `typescript` at the top)

Comment (SgtPooki) [reply packages/verified-fetch/src/index.ts:151]: official github markdown states `typescript` is correct, but i'm not sure that it's case-sensitive.

1. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
1. https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L9-L11
1. https://github.com/github-linguist/linguist/blob/559a6426942abcae16b6d6b328147476432bf6cb/lib/linguist/languages.yml#L7217C3-L7217C11

**Review by SgtPooki — APPROVED** [review]
lgtm

**Review by achingbrain — APPROVED** [review]

**Pull request merged**